src/osd/PrimaryLogPG.cc

```diff
@@ -3824,6 +3824,12 @@ void PrimaryLogPG::do_scan(
       } else {
 	// we canceled backfill for a while due to a too full, and this
 	// is an extra response from a non-too-full peer
```

Comment:
Instead of the complex check, let's just add a single debug line
<pre>
dout(20) << __func__ << " canceled backfill (too full?)" << dendl;
</pre>